client/components/review.jsx

```diff
@@ -15,32 +15,36 @@ const Review = props => (
       userReviews={props.review.userReviews}
     />
     <div className="rev_reviewData">
-      <div className="rev_starsDateBox">
-        <div><Stars rating={props.review.rating} revId={props.review['_id']} /></div> {/* eslint-disable-line */}
-        <div className="rev_date">{moment(props.review.date).format('M/D/YYYY')}</div>
+      <div id="rev_reviewDataTop">
+        <div className="rev_starsDateBox">
+          <div><Stars rating={props.review.rating} revId={props.review['_id']} /></div> {/* eslint-disable-line */}
```

Comment:
Since you use props.review so much i'd consider destructuring

Reply:
Thanks for this suggestion. It's a good one and I'll keep it in mind when I refactor